tools/linker/CoreOptimizeGeneratedCode.cs

```diff
@@ -44,9 +45,34 @@ public abstract class CoreOptimizeGeneratedCode : ExceptionalSubStep {
 		// This is per assembly, so we set it in 'void Process (AssemblyDefinition)'
 		bool InlineIntPtrSize { get; set; }
 
+		bool? is_arm64_calling_convention;
+
 		public CoreOptimizeGeneratedCode (LinkerOptions options)
 		{
 			Options = options;
+
+			if (Optimizations.InlineIsARM64CallingConvention == true) { 
+				if (options.Target.Abis.Count == 1) {
+					// We can only inline Runtime.InlineIsARM64CallingConvention if the generated code will execute on a single architecture (not if building for armv7+armv7s for instance).
```

Comment:
why ? is `armv7+armv7s` then we know arm64 calling conversion is always `false` right ?

Reply:
You're correct, I've changed the logic accordingly.